.github/workflows/build-api-ref.yml

```diff
@@ -0,0 +1,41 @@
+name: Generate API Reference Documentation
```

Comment:
I am curious how did you test it?  Using local Github act? Or you tested it in separate repo?

Reply:
I forked the openAPI repo in my own GitHub and tested it out there, not sure if you can see it but you can see some runs here: https://github.com/cszhu/openapi-clients/actions

Reply:
Nice, I didn't get it myself. Next time I will try this approach.